src/layered_config_tree/__init__.py

```diff
@@ -7,6 +7,15 @@
     __title__,
     __uri__,
 )
+
+__all__ = [
```

Comment:
What does this do?

Reply:
It solves error: `error: Module "layered_config_tree" does not explicitly export attribute "ConfigurationKeyError"  [attr-defined]`

Reply:
My understanding is that this __all__ list limits what actually gets installed if one were to `pip isntall *`. But more to the point it is one of the ways to pass mypy. I can't say I totally understand it.